libs/printing/src/render.tsx

```diff
@@ -180,7 +182,7 @@ export async function renderToPdf(
         height: inchesToText(
           height === Infinity
             ? Math.max(PAPER_DIMENSIONS.Letter.height, contentHeight)
-            : PAPER_DIMENSIONS.Letter.height
```

Comment:
I'm not sure if we meant to hard code this, but I believe this would have broken legal paper size (or any non-11" paper)

Reply:
Seems like the right change to me